app/src/main/java/com/projectit3048c/dao/iFoodDAO.kt

```diff
@@ -5,13 +5,13 @@ import retrofit2.Call
 import retrofit2.http.GET
 import retrofit2.http.Query
 
-interface iFoodDAO {
+interface IFoodDAO {
```

Comment:
This same suggestion appeared in a previously submitted code review.
When that occurs, only the first gets credit.